pinot-core/src/main/java/org/apache/pinot/core/transport/QueryServer.java

```diff
@@ -57,10 +61,11 @@ public class QueryServer {
   private final ServerMetrics _serverMetrics;
   private final TlsConfig _tlsConfig;
   private final AccessControl _accessControl;
+  public static final Logger LOGGER = LoggerFactory.getLogger(QueryServer.class);
```

Comment:
(minor) Make it `private static final` and move it ahead of other non-static fields.